src/Services/UserService.php

```diff
@@ -191,4 +191,17 @@ public function getAll($search = array(), $isAndCondition = true)
 
         return $results;
     }
+
+    /**
+     * @return array id of User
+     */
+    public function getIdByUsername($username)
+    {
+        $id = sqlQuery("SELECT `id` FROM `users` WHERE BINARY `username` = ?", [$username]);
+        if (!empty($id)) {
+            return $id;
+        } else {
+            return false;
+        }
```

Comment:
would consider this instead (since just requesting one item; then will be less awkward when dealing with response when using the function since the return is the id and not in an array):
```
        if (!empty($id['id'])) {
            return $id['id'];
        } else {
            return false;
        }
```